### DISCUSSION TITLE
access to the application through the phone

### DISCUSSION THREAD

**Discussion by Malcak** [leonlatsch/Photok#126] [Q&A] [answered]
If I hide the application like a phone, how can I access the application again?

**Comment by leonlatsch** [comment, marked as answer]
You can access the application again by entering your secret start code in the phone app.

Eg: `*#*#[code]#*#*`

This is also explained in the dialog before hiding the app.

If you didn't change your code it would be `*#*#1337#*#*`

**Reply by Malcak** [reply]
It's true, sorry for not reading before hiding the app, by the way in the default phone app in samsung the code doesn't work but in the google phone app it works perfectly.

Thank you very much :D and thanks for the good app.
